providers/apis/defi/raydium/provider.go

```diff
@@ -0,0 +1,253 @@
+package raydium
```

Comment:
small nit: can we rename the file to `fetcher.go` just so all of our provider implementations reference the underlying interfaces they are implementing?